src/test/java/step1/StringUtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class StringUtilsTest {
+
+    @DisplayName("요구사항 1 : split 테스트")
+    @Test
+    void splitTest(){
+        //given
+        String sentence ="1,2";
+        String sentence2 = "1";
+
+        //when
+        String[] data = sentence.split(",");
+        String[] data2 = sentence2.split(",");
+
+        //then
+        assertThat(data).containsExactly("1","2");
+        assertThat(data2).containsExactly("1");
+    }
+
+    @DisplayName("요구사항 2 : substring 테스트")
+    @Test
+    void subStringTest() {
+        //given
+        String sentence = "(1,2)";
+
+        //when
+        String result = sentence.substring(sentence.indexOf("(") + 1, sentence.indexOf(")"));
+
+        //then
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @DisplayName("요구사항 3 : charAt 테스트")
+    @Test
+    void charAtTest(){
+        //given
+        String sentence = "abc";
+        int index = 10;
+
+        //then //when
+        assertThatThrownBy(() -> { sentence.charAt(index); })
```

Comment:
```suggestion
        assertThatThrownBy(() ->  sentence.charAt(index))
```
이건 개발자마다의 차이인데...
명확하게 하기 위해서 `{}`을 사용할 수 있고, 가독성을 위해서 지울 수도 있어요(세미콜론도 사라지죠) :)  
이부분은 피드백이 아닌 참고만 부탁드릴게요!!